docs/membership/online-membership-sign-up.md

```diff
@@ -27,7 +27,7 @@ contribution page for online sign up (see free memberships below for
 instructions on how to do this).
 
 Contribution pages are very powerful and have a lot of options that are
-grouped together into tabs. Once you have given your contribution page a
+grouped together into tabs; instructions for creating one are [here](https://docs.civicrm.org/user/en/latest/contributions/online-contributions/). Once you have given your contribution page a
```

Comment:
```suggestion
grouped together into tabs.
You may wish to read the [Online Contributions](https://docs.civicrm.org/user/en/latest/contributions/online-contributions/) documentations chapter, which provides instructions for setting up contribution pages. 
Once you have given your contribution page a
```

Reply:
We generally try and avoid links that don't describe themselves - for accessibility reasons mainly. Avoiding links that just have the text "here" or "read" unless we're also adding separate link titles.